File: fizz-buzz-whiz/typescript/src/fizzBuzzWhiz.ts
export function say(n: number): string {
  const divisibleByThree = n % 3 === 0;
  const divisibleByFive = n % 5 === 0;

  if (divisibleByThree && divisibleByFive) return 'FizzBuzz';
  if (divisibleByThree) return 'Fizz';
  if (divisibleByFive) return 'Buzz';
  return String(n);
}

File: fizz-buzz-whiz/python/src/fizz_buzz_whiz/fizz_buzz_whiz.py
def say(n: int) -> str:
    divisible_by_three = n % 3 == 0
    divisible_by_five = n % 5 == 0

    if divisible_by_three and divisible_by_five:
        return "FizzBuzz"
    if divisible_by_three:
        return "Fizz"
    if divisible_by_five:
        return "Buzz"
    return str(n)
